core/io/file_access_pack.cpp

```diff
@@ -36,11 +36,11 @@
 
 #define PACK_VERSION 1
 
-Error PackedData::add_pack(const String &p_path) {
+Error PackedData::add_pack(const String &p_path, bool p_overwrite_current) {
```

Comment:
@reduz suggests to rename the argument `p_replace_files`. Otherwise it should be good to merge.